src/server/trpc.ts

```diff
@@ -0,0 +1,24 @@
+import { User } from '@prisma/client';
+import { initTRPC } from '@trpc/server';
+
+import { db } from "~/server/db";
+
+const t = initTRPC.context<User>().create();
+
+export const router = t.router;
+export const publicProcedure = t.procedure;
+
+const appRouter = router({
+  getById: publicProcedure.query(async (opts) => {
+    const userId = opts.ctx.id;
+    return db.user.findUnique({
```

Comment:
What happens when the user is not found? Can you show what happens when a user is not found and in general how to handle errors like that? tRPC docs will probably be helpful for this